docs/user_guide/representing/array.md

```diff
@@ -0,0 +1,463 @@
+# Collection of documents
+
+DocArray allow users to represent and manipulate multi-modal data to build AI application (Generative AI, neural search ...). 
+DocArray could be seen as a `multi-modal extension of Pydantic for Machine Learning use case`. 
+
+!!! warning
+    DocArray is actually more than just a Pydantic extension, it is a general purpose multi-modal python libraries. 
+     But it is usefully to see it that way to fully understand the representing ability that DocArray offer.
+
+As you have seen in the last section (LINK), the fundamental building block of DocArray is the [`BaseDoc`][docarray.base_doc.doc.BaseDoc] class which allows to represent a *single* document, a *single* datapoint.
+
+In Machine Learning though we often need to work with an *array* of documents, an *array* of datapoints.
+
+This section introduce the concept of `AnyDocArray` LINK which is an (abstract) collection of `BaseDoc`. This library
```

Comment:
Reading this I am starting to have doubts about the name `AnyDocArray`. Should we drop the `Any`?